src/lib.rs

```diff
@@ -88,7 +87,8 @@ struct Bloom {
 /// https://github.com/mozilla/filter-cascade/blob/v0.3.0/filtercascade/fileformats.py
```

Comment:
This should probably be updated to match the release that has the new enum value, or perhaps a version-agnostic link?

Reply:
Agreed, we can update it after https://github.com/mozilla/filter-cascade/pull/26 is merged and filter-cascade gets a version bump.